searchItemName.ts

```diff
@@ -0,0 +1,40 @@
+import sqlite3 from "sqlite3";
+import { itemDBrow } from "./types/interfaces";
+
+const db = new sqlite3.Database("./db/items.db", (error) => {
+    if (error){
+        console.error(error.message);
+    }
+    console.log("Connected to items database");
+});
+
+export async function searchItemName(
+    itemName: string,
+
+): Promise<number> {
+
+    return new Promise((resolve,reject) => {
+        db.get(
+            "SELECT *FROM items WHERE itemName=?",
```

Comment:
Can we format this string a bit better? Also this SQL query returns exact name matches, usually for searches we want to be a bit lenient and not get only exact matches. Ideall we would do a [fuzzy search](https://www.placekey.io/blog/sql-address-matching), but that would take too much time atm, so we can leave it for later. For now, I suggest using a more lenient matching operator with LIKE %name%. [Reference](https://www.w3schools.com/sql/sql_like.asp)